.evergreen/functions.yml

```diff
@@ -629,10 +633,15 @@ functions:
           BROWSER_NAME: ${browser_name}
           E2E_TEST_GROUPS: ${e2e_test_groups}
           E2E_TEST_GROUP: ${e2e_test_group}
+          ATLAS_LOCAL_VERSION: latest
         script: |
           set -e
           # Load environment variables
           eval $(.evergreen/print-compass-env.sh)
+
```

Comment:
For web, these tests are skipped, so no point in setting it up for web. You should move this to other function, where we test packaged app, right?